frontend/src/app/modules/home/projects/create-project/create-project.component.ts

```diff
@@ -0,0 +1,140 @@
+            { label: 'Mobile', command: (event) => this.onTabChange(event) },
+            { label: 'Desktop', command: (event) => this.onTabChange(event) }
+        ];
+        this.activePlatformTabItem = Object.assign(this.platformTabItems[0]);
+        this.selectedAlertType = String(this.alertTypes[0]);
+        this.selectedAlertTimeInterval = String(this.alertTimeIntervals[0]);
+        this.alertsCount = 10;
+        this.formGroup = new FormGroup({
+            alertCategory: new FormControl(
+                this.alertCategories[0],
+                [
+                    Validators.required
+                ]
+            ),
+            projectName: new FormControl(
+                '',
+                [
+                    Validators.required,
+                    Validators.minLength(3),
+                    Validators.maxLength(50),
+                    Validators.pattern('^[a-zA-Z0-9-_]+$')
+                ]
+            ),
+            team: new FormControl(
+                this.userTeams[0],
+                [
+                    Validators.required
+                ],
+            ),
+        });
+        this.onTabChange();
+    }
+
+    initFakeData() {
+        this.platformCards = this.fakeData.fakePlatforms;
+        this.alertTypes = this.fakeData.fakeAlertTypes;
+        this.alertTimeIntervals = this.fakeData.fakeAlertTimeIntervals;
+        this.alertCategories = this.fakeData.fakeAlertCategories;
+        this.userTeams = this.fakeData.fakeTeams;
+    }
+
+    onTabChange(event?: any): void {
+        this.selectedPlatformId = undefined;
+        switch (event?.item.label ?? '') {
+            case 'Browser': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isBrowser === true)];
+                break;
+            }
+            case 'Server': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isServer === true)];
+                break;
+            }
+            case 'Mobile': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isMobile === true)];
+                break;
+            }
+            case 'Desktop': {
+                this.viewPlatformCards = [...this.platformCards.filter(value => value.isDesktop === true)];
+                break;
+            }
+            default: {
+                this.viewPlatformCards = [...this.platformCards];
+                break;
+            }
+        }
+    }
+
+    onPlatformSelected(platformId: number): void {
+        this.selectedPlatformId = platformId === this.selectedPlatformId ? undefined : platformId;
+    }
+
+    applySelection(platformId: number) {
+        return platformId === this.selectedPlatformId ? { 'selected-card': true } : { 'selected-card': false };
+    }
+
+    createProject(): void {
+        if (this.formGroup.valid && this.selectedPlatformId !== undefined) {
```

Comment:
You can just write this.selectedPlatformId instead of this.selectedPlatformId !== undefined

Reply:
![image](https://user-images.githubusercontent.com/36418695/128164574-01fe0332-b0e0-43f5-a72a-3bae0956b864.png)

Reply:
any problems here?